From 5330a3d65b7877f5f3691c20df35fbb7856d6f38 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Thu, 11 Jan 2018 21:28:15 +0100
Subject: [PATCH] Fix Qt5LinguistTools detection/lrelease binary location

Correctly find the Qt5 module that provides the path to Qt5-based lrelease.
Available since >= Qt-5.3.1 which is well below the current minimum version.
---
 src/translations/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/translations/CMakeLists.txt b/src/translations/CMakeLists.txt
index d9afded..174acea 100644
--- a/src/translations/CMakeLists.txt
+++ b/src/translations/CMakeLists.txt
@@ -1,7 +1,8 @@
 # translations for 'simplescreenrecorder' executable
 
 if(WITH_QT5)
-	find_program(LRELEASE NAMES lrelease-qt5 lrelease)
+	find_package(Qt5LinguistTools REQUIRED)
+	set(LRELEASE Qt5::lrelease)
 else()
 	find_program(LRELEASE NAMES lrelease-qt4 lrelease)
 endif()
-- 
2.29.2

